lib/AstOfLlbc.ml

```diff
@@ -2376,7 +2389,7 @@ let file_of_crate (crate : Charon.LlbcAst.crate) : Krml.Ast.file =
   } =
     crate
   in
-  if options.remove_associated_types <> [ "*" ] then begin
+  if options.preset <> Some (Eurydice) then begin
```

Comment:
@Nadrieril ultimately this was the root cause -- the behavior changed between symcrust and libcrux because the former was not passing --preset=eurydice, and the comment below was updated but not the runtime check

Reply:
I don't understand, the preset implies the `remove_associated_types` option, how could this have changed anything?

Reply:
I kept the runtime check the same on purpose to avoid breaking libcrux, this was a forward-compatible change to avoid disruption.